### DISCUSSION TITLE
What is the process for smaller bugs?

### DISCUSSION THREAD

**Discussion by PatricioNG** [prisma/prisma#23074] [Q&A]
### Question

Wondering what the dev lifecycle looks like for smaller bugs at Prisma? Is there a regular period of the year where they are looked at or is it just based off of scale of impact? I've asked in a few different threads on Slack, as well as commenting on the issue but haven't heard anything back yet. I've been loving Prisma since 2020 and have integrated it into many core enterprise apps at work, most recently however we made the transition from MySQL to SQL Server and have run into a handful of bugs for SQL Server. 

For the most part, the bugs don't get in the way and we can work around them, but there's one experience-breaking bug that unfortunately is plaguing us very heavily https://github.com/prisma/prisma/issues/21778 - the chunking logic is broken for SQL Server and we're finding that it's been bringing down production apps where we didn't expect. In the past usage was smaller but as we scale apps up we keep running into new crashes as logic that was never under 2100 parameters now starts to cross the threshold and cause errors to be thrown. We can patch it when it comes up, but the patches often are ugly workarounds and inefficient.

We're starting to use Postgres in a few of our apps as well (plan is to have SQL Server and Postgres depending on the app) and my goal is to eventually try to bring Pulse into our applications but there's been negative feedback about the chunking bug that's eating into a business case so hoping to get an understanding of if there's a timeline to resolve so I can guide the conversation. Thanks for any insight! Looking forward to keeping the Prisma train rolling 🚂

**Comment by ludralph** [comment]
Hi @PatricioNG 👋 

Thank you for your efforts in providing a reproduction of the issue. 

> Is there a regular period of the year where they are looked at or is it just based off of scale of impact? 

We do have a bug reproduction day and based on our initial assessment of the problem description, we anticipate that reproducing it is feasible. Once we have successfully replicated the issue, we will be better positioned to address and potentially resolve it as soon as possible.

**Reply by PatricioNG** [reply]
Thanks @ludralph, it would be good to understand what the timelines at Prisma look like 😌 I did attach a full reproduction to the ticket in a repo, so here's hoping people aren't spending time from scratch and leveraging the community

**Reply by ludralph** [reply]
Hi @PatricioNG 

We recently released the [Prisma ORM Manfiesto](https://github.com/prisma/prisma/discussions/25772) that will help set expectations for how we approach things moving forward. Please take a look.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;
